### PR TITLE
[C-2926] Implement selected values for upload contextual menu fields

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds selected values for each of the upload contextual menu fields.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2926-selected-value

File: packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx [hunk 1]
```diff
@@ -11,7 +10,7 @@ import {
 } from 'components/form-fields'
 import { moodMap } from 'utils/Moods'
 
-import { getTrackFieldName } from '../forms/utils'
+import { getTrackFieldName } from '../hooks'
```
Comment (amendelsohn) [inline packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx:13]: I know I did this, but feels weird to import a regular function from a hooks file 🤔

**Review by amendelsohn — APPROVED** [review]
Really nice! Love the renderValue pattern!

File: packages/web/src/components/Icon/Icon.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ export const Icon = (props: IconProps) => {
   const style = color
     ? {
         ...styleProp,
-        '--icon-color': toCSSVariableName(color)
+        '--icon-color': `var(${toCSSVariableName(color)})`
```
[the remark this answers is not in the document]
Comment (dylanjeffers) [reply packages/web/src/components/Icon/Icon.tsx:44]: yeah fairly tragic, lol

File: packages/web/src/pages/upload-page/components/EditPageNew.tsx [hunk 3]
```diff
@@ -90,8 +95,14 @@ export const EditPageNew = (props: EditPageProps) => {
...
-              <TrackMetadataFields playing={false} />
-              <TrackModalArray />
+              <TrackMetadataFields />
+              <div className={styles.advancedOptions}>
+                <ReleaseDateField />
+                <RemixSettingsField />
+                <SourceFilesField />
+                <AccessAndSaleField />
+                <AttributionField />
+              </div>
```
[the remark this answers is not in the document]
Comment (dylanjeffers) [reply packages/web/src/pages/upload-page/components/EditPageNew.tsx:105]: yeah agreed id like a better name, something we can consider with product team during our qa runthroughs imo, make sure we understand each section and what its intent is

File: packages/web/src/pages/upload-page/fields/SourceFilesField.tsx [hunk 4]
```diff
@@ -85,35 +88,39 @@ export const SourceFilesModalForm = () => {
...
-    <div className={styles.preview}>
-      <div className={styles.header}>
-        <Text className={styles.title} variant='title' size='large'>
-          {messages.title}
-        </Text>
-      </div>
-      <Text>{messages.description}</Text>
-    </div>
-  )
+  const renderValue = () => {
```
[the remark this answers is not in the document]
Comment (dylanjeffers) [reply packages/web/src/pages/upload-page/fields/SourceFilesField.tsx:91]: yeah defintely agree with thoughts here. "value" was chosen cause each field has a value like you said, but could see using language like "renderSelectedLabels" which would be in line with what the mockups are saying. That would require changing a bunch of the component names too which im totally down. for also. ill make a tech-debt story to start collecting these great ideas.

**Pull request merged**